internal/docstore/driver/util.go

```diff
@@ -14,8 +14,31 @@
 
 package driver
 
-import "github.com/google/uuid"
+import (
+	"math/rand"
+	"sync"
+
+	"github.com/google/uuid"
+)
 
 // UniqueString generates a string that is unique with high probability.
 // Driver implementations can use it to generate keys for Create actions.
 func UniqueString() string { return uuid.New().String() }
+
+// Call when running tests that will be replayed.
+// Each seed value will result in UniqueString producing the same sequence of values.
+func MakeUniqueStringDeterministicForTesting(seed int64) {
```

Comment:
These functions seem can be moved to drivertest package.

Reply:
Done.